pipeline_inspection_metacontrol/pipeline_inspection_metacontrol/follow_pipeline_lc.py

```diff
@@ -80,18 +82,25 @@ def mission(self):
         for gz_pose in pipe_path.result().path.poses:
             setpoint = self.ardusub.setpoint_position_gz(
                 gz_pose, fixed_altitude=True)
-            while not self.ardusub.check_setpoint_reached(setpoint, 0.2):
+            count = 0
```

Comment:
Are you still going to move this to the check_setpoint_reached method?

Reply:
Not really, for now I will just leave it like this since it is easier.